src/msg/kava/index.ts

```diff
@@ -154,111 +193,137 @@ function newMsgSubmitProposal(proposal, proposer, committeeID) {
   };
 }
 
-function newMsgVote(proposalID, voter, voteType) {
+function newMsgVote(proposalID: string, voter: string, voteType: VoteType) {
   return {
     type: 'kava/MsgVote',
     value: {
       proposal_id: String(proposalID),
       voter: voter,
-      vote_type: voteType
+      vote_type: voteType,
     },
   };
 }
 
+function newMsgVoteGovernance(proposalID: string, voter: string, voteType: VoteType) {
```

Comment:
do we want to move this on the the `src/msg/cosmos/index.ts` file since it's not kava specific?

Reply:
ah, sure